setup-homebrew/action.yml

```diff
@@ -10,5 +10,9 @@ outputs:
   gem-hash:
     description: Homebrew's Ruby Gemfile.lock sha256sum
 runs:
-  using: node12
-  main: main.js
+  using: composite
+  steps:
+    - run: chmod +x $GITHUB_ACTION_PATH/main.sh
```

Comment:
Will this show up in the output? Is it avoidable by doing `chmod +x` on the file in this repository? Alternatively, could do `bash $GITHUB_ACTION_PATH/main.sh` maybe?

Reply:
I did `chmod +x main.sh` locally and committed it.